### PR TITLE
Start Ruby section with installation instructions

### PR DESCRIPTION
**Pull request by monfresh**
**Why**: Some of our client projects use Ruby, so it's important to
build out the Ruby section of our playbook to establish conventions
and best practices.

### CODE REVIEW

File: developing/languages/ruby/installation.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+* [bundler](https://bundler.io/)
+
+The script also updates your shell startup file (such as `.bash_profile`) to
+properly configure `chruby`.
+
+[script]: https://github.com/monfresh/install-ruby-on-macos
+
+### Docker installation
+
+WIP
```
Comment (ahobson) [inline developing/languages/ruby/installation.md]: I vote either not including this section or putting an example of

`docker run -it --rm=true -v $PWD:$PWD -w $PWD ruby:${DESIRED_RUBY_VERSION}-slim bash`

Comment (monfresh) [reply developing/languages/ruby/installation.md]: I expanded the Docker section here: https://github.com/trussworks/Engineering-Playbook/pull/43/commits/c694925782d00632b69e0699701b6b941993dd08

Thoughts?

Comment (ahobson) [reply developing/languages/ruby/installation.md]: Awesome!  We do have a docker section in the handbook, maybe link to that for the installation instructions?

Comment (monfresh) [reply developing/languages/ruby/installation.md]: Great idea! I missed that.

Comment (monfresh) [reply developing/languages/ruby/installation.md]: Fixed here: https://github.com/trussworks/Engineering-Playbook/pull/43/commits/7d2213e27b75330b77b19c7d7f47a2c9e917d67f

File: developing/languages/ruby/installation.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+2. `cd` into the directory of the Ruby project you want to work with
+3. Run a bash shell in Docker: `docker run -it --rm=true -v $PWD:$PWD -w $PWD ruby:${DESIRED_RUBY_VERSION}-slim bash`
+
+For example, if you want an image with Ruby 2.6.4, you would run:
+
+```
+docker run -it --rm=true -v $PWD:$PWD -w $PWD ruby:2.6.4-slim bash
+```
+
+### Ruby managers
```
Comment (pkarman) [inline developing/languages/ruby/installation.md]: I like this section.

**Pull request merged**